training/vision/components/image_classification/spec.yaml

```diff
@@ -5,72 +5,168 @@ description: Trains an image classification model. [Learn More](https://aka.ms/b
 
 name: train_image_classification_model
 display_name: Train Image Classification Model (Private Preview)
-version: 0.0.2-preview
+version: 0.0.1
 
 inputs:
   training_data:
     type: mltable
-    description: Training data
+    description: Path to MLTable for training data.
   validation_data:
     type: mltable
     optional: true
-    description: Validation data
+    description: Path to MLTable for validation data.
+  task_type:
+    description: Whether a single image can have multiple labels.
+    type: string
+    enum: ['image-classification', 'image-classification-multilabel']
```

Comment:
nit: may want to have tasks without hyphens displayed to the user? no idea, just thinking out loud

Reply:
JOS/backend will send out these existing task types... also current users sdk/cli should be familiar with these types... so wanted to use them as is..